docs/SERVICE_AVAILABILITY.md

```diff
@@ -182,17 +182,10 @@ These services provide a way to access content from any business unit from any d
 
 ⚠️ Pagination is supported, but with a limit of 50. Attempting to request larger page sizes will fail.
 
-## Image scaling services
+## Image scaling service
 
-A ⚠️ means that a service is supported, but might not return an image with the exact requested dimension.
+Only prefined width or size are available for scaling.
 
-| Request | SRF | RTS | RSI | RTR | SWI |
-|:-- |:--:|:--:|:--:|:--:|:--:|
-| Video image with width | ✅ | ✅ | ⚠️ | ✅ | ✅ |
-| Video image with height | ✅ | ✅ | ❌| ✅ | ⚠️ |
-| Audio image with width | ✅ | ✅ | ⚠️ | ✅ | ❌ |
-| Audio image with height | ⚠️ | ✅ | ❌ | ⚠️ | ❌ |
+The [PlaySRG image service](https://confluence.srg.beecollaboration.com/display/SRGPLAY/Project+-+Image+Service) is used for all image urls*.
 
-### Remark
-
-Scale image URL services are BUs dependant
+*An exception exists when the url host contains `rts.ch` and the url path contains `.image`. In this case, the business unit scaling image service is still used, using the convention of `https:[BU].ch/[image path]/scale/width/[width]`.
```

Comment:
Not sure this implementation detail should be exposed in the documentation. It should (hopefully) disappear in the future anyway.

Reply:
@defagos I agree. It was written when no unit tests existed. Then  `ImageURLsTestCase.m` added.
I would not recommend to hide the exception, but simplify the documentation.

Proposition:

> *An exception exists for some `rts.ch` image urls. In this case, the business unit scaling image service is still used, using the convention of `https:[BU].ch/[image path]/scale/width/[width]`.

Reply:
@defagos Moved to a dedicated issue. Is it better?